src/app/drive/components/DriveExplorer/DriveExplorerList/DriveExplorerList.tsx

```diff
@@ -423,8 +417,7 @@ const DriveExplorerList: React.FC<DriveExplorerListProps> = memo((props) => {
             onRKeyPressed: () => {
               if (props.selectedItems.length === 1) {
                 const selectedItem = props.selectedItems[0];
-                dispatch(uiActions.setCurrentEditingNameDirty(selectedItem.name));
-                dispatch(uiActions.setCurrentEditingNameDriveItem(selectedItem));
+                renameItem(selectedItem);
```

Comment:
When the user clicks the "R" key on the keyboard it gets typed on the rename dialog (use e.preventDefaults() when pressing a hotkey)